scripts/data_files/query_config.fmt

```diff
@@ -99,6 +99,10 @@
 #define MACRO_NAME_TO_STR(macro)                                        \
     mbedtls_printf( "%s", strlen( #macro "" ) > 0 ? #macro "\n" : "" )
 
+#define NAME_TO_STR(macro)  #macro
+#define OUTPUT_MACRO_NAME_VALUE(macro) mbedtls_printf( #macro "%s\n",   \
+    strlen( NAME_TO_STR(macro) "") > 0 ? "=" NAME_TO_STR(macro) : "" )
```

Comment:
Minor: why `NAME_TO_STR(macro) ""` and not just `NAME_TO_STR(macro)`?

Reply:
Changed like below comments